libs/vips/Makefile

```diff
@@ -8,49 +8,65 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vips
-PKG_VERSION:=8.6.5
+PKG_VERSION:=8.7.2
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://github.com/jcupitt/libvips/releases/download/v8.6.5/
-PKG_HASH:=8702af0e340e220e0c08f8ded6c8248b18e7043938d9e8a2426631fd37a9d5db
+PKG_SOURCE_URL:=https://github.com/libvips/libvips/releases/download/v$(PKG_VERSION)
+PKG_HASH:=66c8d1f1db1d0934091e101abe8b3de552f45fb03d120dc10aa15450058d4baa
+
+PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
+PKG_LICENSE:=LGPL-2.1+
+PKG_LICENSE_FILES:=COPYING
+
 PKG_FIXUP:=autoreconf
-PKG_CHECK_FORMAT_SECURITY:=0
+PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/nls.mk
 
-PKG_INSTALL=1
-
 define Package/vips
   $(call Package/vips/Default)
   SECTION:=multimedia
   CATEGORY:=Multimedia
   TITLE:=An image manipulation library
-  URL:=http://www.vips.ecs.soton.ac.uk/
-  MAINTAINER:=W. Michael Petullo <mike@flyn.org>
+  URL:=https://libvips.github.io/libvips/
   DEPENDS:=+glib2 +libexif +libexpat +libjpeg +libpng +giflib +libstdcpp +libxml2 $(INTL_DEPENDS)
 endef
 
 TARGET_LDFLAGS+= \
 	-Wl,-rpath-link=$(STAGING_DIR)/usr/lib
 
 CONFIGURE_ARGS += \
-	--disable-cxx \
-	--disable-threads \
+	--disable-debug \
+#	--disable-deprecated \ currently breaks the build
```

Comment:
This is the line causing the mysterious **\*\*\* recipe commences before first target. Stop.** error. Just delete it or move the comment out of the CONFIGURE_ARGS line. (Or, move the \ to the end-of-line where it might work.)

Make is pretty picky about spaces/tabs/indentation.